app/views/shared/_navbar.html.erb

```diff
@@ -40,7 +40,7 @@
           </div>
 
           <%= link_to "See the Arts", arts_path, class: "option" %>
-          <%= link_to "Create a listing", new_art_path, class: "option" %>
+          <%= link_to 'Create an art listing', "/arts/building/build", data: { "turbo-method": :post }, class: "option" %>
```

Comment:
"Create an art listing" is kind of long and will clash with the rest of the links. "Create a listing" already kinda was.
What do you think of "Share new art"